### DISCUSSION TITLE
missing file

### DISCUSSION THREAD

**Discussion by uniqfox** [HclX/WyzeHacks#156] [General]
wyze_updater.py was absent in the zip archive fyi

**Comment by kejames69** [comment]
python3: can't open file '/home/ken/WyzeHacks/installer/./wyze_updater.py': [Errno 2] No such file or directory

**Reply by LeftHandRev** [reply]
I am getting this error, too. A symlink to file is in fact there, but it has no target. Per @uniqfox's claim, the file is missing.

**Reply by jjbudd3** [reply]
I am unable to get past this error. Does this script still work?
